src/parse-tsconfig/index.ts

```diff
@@ -291,7 +291,6 @@ const normalizeCompilerOptions = (
 
 		if (target === 'esnext') {
 			compilerOptions.module ??= 'es6';
```

Comment:
We've set default value of `module` here, and default value of `moduleResolution` will be set after line 321
https://github.com/sxzz/get-tsconfig/blob/ef1aad00c28d4ba6b8494204f468b510567e433c/src/parse-tsconfig/index.ts#L363-L376